sesman/sesman.c

```diff
@@ -413,6 +413,12 @@ main(int argc, char **argv)
         g_file_close(fd);
     }
 
+    if (session_init_shared())
```

Comment:
Please always add brackets for if body statements. Check out xrdp coding style in wiki.